hbase-common/src/main/java/org/apache/hadoop/hbase/CellUtil.java

```diff
@@ -714,39 +418,13 @@ public boolean advance() {
     };
   }
 
-  /**
-   * @param left
-   * @param right
-   * @return True if the rows in <code>left</code> and <code>right</code> Cells match
-   * @deprecated As of release 2.0.0, this will be removed in HBase 3.0.0.
-   *             Instead use {@link #matchingRows(Cell, Cell)}
-   */
-  @Deprecated
-  public static boolean matchingRow(final Cell left, final Cell right) {
-    return matchingRows(left, right);
-  }
-
-  /**
-   *  @deprecated As of release 2.0.0, this will be removed in HBase 3.0.0.
-   *             Instead use {@link #matchingRows(Cell, byte[])}
-   */
-  @Deprecated
-  public static boolean matchingRow(final Cell left, final byte[] buf) {
-    return matchingRows(left, buf);
-  }
-
   public static boolean matchingRows(final Cell left, final byte[] buf) {
     if (buf == null) {
       return left.getRowLength() == 0;
     }
     return PrivateCellUtil.matchingRows(left, buf, 0, buf.length);
   }
 
-  public static boolean matchingRow(final Cell left, final byte[] buf, final int offset,
```

Comment:
This is not deprecated?

Reply:
It is ok to remove this?

Reply:
It is fine to remove this, not being used anywhere

Reply:
CellUtil is IA.Public so client applications might depend on this method.

Reply:
Oh sorry, missed it, let me revert now

Reply:
Done @saintstack @petersomogyi